### DISCUSSION TITLE
Maps is not showing

### DISCUSSION THREAD

**Discussion by TechKayaroll** [visgl/react-google-maps#269] [General]
Hi can someone help me? 
i would like to create a Map Component that shows a map based on props (Lat, Long, Radius). 

And I see geometry example. i tried it on codesandbox but it doesn't work like i expected.

What i did:
1. Replace the API_KEY with my GoogleMaps API Key.
2. I see Marker deprecated warning, but it just a warning so i ignore it.
3. I remove LatLngEquals because there's no exported function, that cause error on Circle Component.
```
  useEffect(() => {
    if (!center) return;
    circle.setCenter(center);
  }, [center]);
```

Question:
1. Why the map still not showing?

here's my codesandbox:
https://codesandbox.io/p/devbox/little-sky-l89ztm

![image](https://github.com/visgl/react-google-maps/assets/146938125/0807e77e-cc6a-43ec-8a69-6576dc73eee9)

**Comment by mrMetalWood** [comment]
It could be that there is an old version of `@vis.gl/react-google-maps` cached on codesandbox somehow since in the examples we have this `"@vis.gl/react-google-maps": "*"` in the package.json 🤔 

could you try to hit the "Update to latest" in codesandbox for the package? And then restart the server
![Screenshot 2024-03-20 at 09 24 03](https://github.com/visgl/react-google-maps/assets/2902795/6d12ae1e-d6b9-48ab-a7d5-2e3668438dc2)
